Userland/Libraries/LibWeb/CSS/StyleValues/CalculatedStyleValue.cpp

```diff
@@ -38,7 +39,13 @@ static double resolve_value(CalculatedStyleValue::CalculationResult::Value value
         [](Number const& number) { return number.value(); },
         [](Angle const& angle) { return angle.to_degrees(); },
         [](Frequency const& frequency) { return frequency.to_hertz(); },
-        [&context](Length const& length) { return length.to_px(*context).to_double(); },
+        [&context](Length const& length) {
+            if (!context.has_value()) {
+                dbgln_if(LIBWEB_CSS_DEBUG, "Length value has no context, returning 0.0");
+                return 0.0;
```

Comment:
This feels like it's covering up a bug elsewhere. Why are we trying to resolve a length without a resolution context?

Reply:
Theres lots of places where we need to pass the context, and currently dont. but for now ill leave this change out and we can slowly fix those places.